test/metabase/query_processor_test/date_time_zone_functions_test.clj

```diff
@@ -140,7 +140,54 @@
                                           :fields      (into [] (for [op ops] [:expression (name op)]))})
                       (mt/formatted-rows (repeat int))
                       first
-                      (zipmap ops)))))))))
+                      (zipmap ops)))))))
+
+    (testing "with timestamptz columns"
+      (mt/test-drivers (filter mt/supports-timestamptz-type? (mt/normal-drivers-with-feature :temporal-extract))
+        (mt/with-report-timezone-id "Asia/Kabul"
+          (is (= (if (or (= driver/*driver* :sqlserver)
+                         (driver/supports? driver/*driver* :set-timezone))
+                     {:get-year        2004,
+                      :get-quarter     1,
+                      :get-month       1,
+                      :get-day         1,
+                      :get-day-of-week 5,
+                      ;; TIMEZONE FIXME these drivers are returning the extracted hours in
+                      ;; the timezone that they were inserted in
+                      ;; maybe they need explicit convert-timezone to the report-tz before extraction?
+                      :get-hour        (case driver/*driver*
+                                         (:sqlserver :presto :presto-jdbc :snowflake :oracle) 5
+                                         2),
+                      :get-minute      (case driver/*driver*
+                                         (:sqlserver :presto :presto-jdbc :snowflake :oracle) 19
+                                         49),
```

Comment:
@qnkhuat Do you think it's worth drafting an issue to get this fixed? I feel this comment will be confusing once the context is lost and how it relates to temporal bucketing, filtering on dates etc

Reply:
I created an issue here: https://github.com/metabase/metabase/issues/26947